.github/workflows/validate.yml

```diff
@@ -132,7 +123,7 @@ jobs:
       - name: Tar cabal head executable
         if: matrix.ghc == env.GHC_FOR_RELEASE
         run: |
-          CABAL_EXEC=$(cabal-plan list-bin --builddir=dist-newstyle-validate-ghc-${{ matrix.ghc }} cabal-install:exe:cabal)
+          CABAL_EXEC=$(cabal list-bin --builddir=dist-newstyle-validate-ghc-${{ matrix.ghc }} --project-file=cabal.project.validate cabal-install:exe:cabal)
           # We have to tar the executable to preserve executable permissions
           # see https://github.com/actions/upload-artifact/issues/38
           if [[ ${{ runner.os }} == 'Windows' ]]; then
```

Comment:
`cabal-plan` on the line below. just in case, try grepping for `cabal-plan` so we don't miss any

Reply:
Done. I removed it in validate.{sh,yml} and Makefile, and i didn't touch couple other places, which don't seem to be related to CI necessarily. There're Dockerfiles but I don't know what to do about them: i have a feeling they have bitrotted. Anyway, I'd prefer to not get involved with them.